### ISSUE TITLE
Week 3 Day 4 - Hackathon - 

### ISSUE THREAD

**Issue by mdewey** [JamesEnoch89/assignments#12]
# Hackathon!

Your homework this weekend is the Hack-a-thon. Good Luck! I am here if you have any questions. 

### Turning in
To turn this in, on Sunday, submit to me the github repo that you are working on.

**Comment by JamesEnoch89** [comment]
didn't get much feedback in the slack channel on what we needed to complete for the bot. 

but airtable has a free two week pro account which is good enough for us to store information in a database using forms and good enough to let remote volunteers contact those in need who sign up using Twilio. Both of which work right now. I just texted myself using the Twilio number. So this might help in the demo.

here is the very basic form preview: https://airtable.com/shrFkvTuywiIjIEPM

and the very basic database preview: 
https://airtable.com/shrdf0QVu7RPyCS9r
